lib/lighthouse/benefits_documents/service.rb

```diff
@@ -64,24 +49,38 @@ def cleanup_after_upload
 
     def submit_document(file, file_params, lighthouse_client_id = nil)
       user_icn = @user.icn
+      user_account_uuid = @user.user_account_uuid
       document_data = build_lh_doc(file, file_params)
+      claim_id = file_params[:claimId] || file_params[:claim_id]
+      tracked_item_id = file_params[:trackedItemIds] || file_params[:tracked_item_ids]
+
+      unless claim_id
+        raise Common::Exceptions::InternalServerError,
+              ArgumentError.new("Claim with id #{claim_id} not found")
+      end
```

Comment:
Wouldn't `#{claim_id}` be empty here so adding it to the log message would just be a empty string?

Reply:
Yup, good catch. I'll fix that. Also, correct about it being inherited code 😆

Reply:
@shaunburdick This error has been changed to say that a claim id is required without printing a claim id in the error